controllers/host/host_controller.go

```diff
@@ -1838,6 +1830,85 @@ func (r *HostReconciler) ReconcileDeletedHost(client *gophercloud.ServiceClient,
 	return nil
 }
 
+func (r *HostReconciler) LockUnlockAnnotationExists(instance *starlingxv1.Host) bool {
+	_, ok := instance.Annotations[cloudManager.LockAndUnlockController]
+
+	return ok
+}
+
+func (r *HostReconciler) ProcessLockHost(host_instance *starlingxv1.Host, host_obj *hosts.Host) error {
+	mon_err, req_success := r.LockHostRequest(host_instance, host_obj.ID, host_obj.Personality, "oam platform network")
```

Comment:
Confused here:
1 isn't "oam platform network" a part of "platform networks"? Can we have constants to clarify the "reconcile_object" here?
2 why don't we check the req_sucess previously